raiden/src/raiden.ts

```diff
@@ -216,6 +217,9 @@ export class Raiden {
 
     // initialize epics, this will start monitoring previous token networks and open channels
     this.store.dispatch(raidenInit());
+    
+    // Patch provider's sign method (https://github.com/raiden-network/light-client/issues/223)
+    patchSignSend(this.provider);
```

Comment:
Please, patch in the `create` static method, right below provider authentication, instead of here. constructor is expected to receive a _compatible_ provider, and this is a workaround is for ethers specific cases. If a user can already provide the compatible provider by theirselves, it's supported for them to bypass this factory static method and instantiate directly.